### ISSUE TITLE
Can't see text

### ISSUE THREAD

**Issue by b-stone** [b-stone/Visualize-M2#102]
@vlandham, can you help me? I can't seem to make the text appear in the example in the following directory:

```
Visualize-M2/Visualize/templates/visSimplicialComplex/test/
```

You can only view it on the `scomplex-b` branch. I know the problem is that I have `<text>` inside of `<circle>` in the DOM. But I can't seem to change it. Here is a screen shot.

![image](https://cloud.githubusercontent.com/assets/1364138/15556608/989293f2-22c6-11e6-9108-040acec2ec35.png)

**Comment by vlandham** [comment]
sure let me take a look

**Comment by bbarwick** [comment]
Labels for 2d simplicial complex vertices have been implemented in pull request #105.

**Comment by vlandham** [comment]
easy peasy. glad i could help (sorry for not really helping at all).
